resources/assets/js/components/HRPortal/JobReviewHrPage.tsx

```diff
@@ -0,0 +1,178 @@
+import React, { useEffect } from "react";
+import {
+  useIntl,
+  FormattedMessage,
+} from "react-intl";
+import { useDispatch, useSelector } from "react-redux";
+import ReactDOM from "react-dom";
+import RootContainer from "../RootContainer";
+import {
+  Job,
+  JobPosterKeyTask,
+  Criteria,
+  Skill,
+  Manager,
+  Department,
+} from "../../models/types";
+import {
+  hrJobSummary,
+} from "../../helpers/routes";
+import { RootState } from "../../store/store";
+import {
+  getJob,
+  getTasksByJob,
+  getCriteriaByJob,
+} from "../../store/Job/jobSelector";
+import { getSkills } from "../../store/Skill/skillSelector";
+import {
+  fetchJob,
+  fetchJobTasks,
+  fetchCriteria,
+} from "../../store/Job/jobActions";
+import { getDepartments as fetchDepartments } from "../../store/Department/deptActions";
+import { getDepartments } from "../../store/Department/deptSelector";
+import {
+  getManagerById,
+} from "../../store/Manager/managerSelector";
+import {
+  fetchManager,
+} from "../../store/Manager/managerActions";
+import { JobReviewDisplay } from "../JobBuilder/Review/JobReview";
+import { fetchSkills } from "../../store/Skill/skillActions";
+
+interface JobReviewHrPageProps {
+  jobId: number;
+  job: Job | null;
+  skills: Skill[];
+  keyTasks: JobPosterKeyTask[];
+  criteria: Criteria[];
+  departments: Department[];
+  manager: Manager | null;
+}
+
+const JobReviewHrPage: React.FunctionComponent<JobReviewHrPageProps> = ({
+  jobId,
+  job,
+  skills,
+  keyTasks,
+  criteria,
+  departments,
+  manager,
+}): React.ReactElement => {
+  const { locale } = useIntl();
+  if (locale !== "en" && locale !== "fr") {
+    throw new Error("Unexpected locale");
+  }
+  return (
+    <div data-c-container="form" data-c-padding="top(triple) bottom(triple)">
+      {job !== null ? (
+        <JobReviewDisplay
+          job={job}
+          manager={manager}
+          tasks={keyTasks}
+          criteria={criteria}
+          skills={skills}
+          departments={departments}
+          hideBuilderLinks={true}
+        />
+      ) : (
+        <div data-c-alignment="base(centre)">
+          <i className="fa fa-spinner fa-spin" />
+        </div>
```

Comment:
This icon should meet accessible standards. I can do this real quick though. I reference this https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility

Reply:
Oh, thanks! Maybe we should build a reusable loading spinner component. Would make it easier to customize consistently too.

Reply:
Ya that's a good idea. Should I make a new issue or did you want to build one on this branch?